packages/mattermost-redux/src/actions/users.ts

```diff
@@ -219,32 +127,45 @@ export function loadMe(): ActionFunc {
             Client4.attachDevice(deviceId);
         }
 
-        const promises = [
-            dispatch(getMe()),
-            dispatch(getMyPreferences()),
-            dispatch(getMyTeams()),
-            dispatch(getMyTeamMembers()),
-        ];
-
         // Sometimes the server version is set in one or the other
         const serverVersion = Client4.getServerVersion() || getState().entities.general.serverVersion;
         dispatch(setServerVersion(serverVersion));
 
-        await Promise.all(promises);
+        const isCollapsedThreads = isCollapsedThreadsEnabled(getState());
+
+        try {
+            await Promise.all([
+                dispatch(getMe()),
+                dispatch(getMyPreferences()),
+                dispatch(getMyTeams()),
+                dispatch(getMyTeamMembers()),
+            ]);
+
+            await dispatch(getMyTeamUnreads(isCollapsedThreads));
+        } catch (error) {
+            if (isLoggedIn) {
+                dispatch({type: UserTypes.LOGIN_FAILURE, error});
+            }
+
+            dispatch(logError(error as ServerError));
 
-        const collapsedReplies = isCollapsedThreadsEnabled(getState());
-        dispatch(getMyTeamUnreads(collapsedReplies));
+            return {error};
+        }
 
         const {currentUserId} = getState().entities.users;
-        const user = getState().entities.users.profiles[currentUserId];
         if (currentUserId) {
             Client4.setUserId(currentUserId);
         }
 
+        const user = getState().entities.users.profiles[currentUserId];
         if (user) {
             Client4.setUserRoles(user.roles);
         }
 
+        if (isLoggedIn) {
+            dispatch({type: UserTypes.LOGIN_SUCCESS, data: null});
```

Comment:
Just in case it matters, we previously passed no data field for the LOGIN_SUCCESS and now we're passing null. I'm not sure if that ends up changing anything, but it could act handle those differently

Reply:
removed the data null key